documentation/release_notes.markdown

```diff
@@ -4,6 +4,73 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.0.8
+
+In addition to several bug fixes, this release adds new HTTP client timeout settings, a special logfile to capture only HTTP traffic, and a JRuby tuning guide to help you get the best performance from Puppet Server.
+
+###New Features
+
+#### Added new http-client timeout settings
+
+We've exposed two new HTTP client timeout settings, for idle and connection timeouts.
+
+* [SERVER-449](https://tickets.puppetlabs.com/browse/SERVER-449) - Expose http-client timeouts from Puppet Server http_connect_timeout and http_read_timeout.
```

Comment:
Not sure if this is critical to cover here, but the names of the configuration settings are `idle-timeout-milliseconds` and `connect-timeout-milliseconds` and can be configured under the `http-client` section of the puppetserver.conf file.  See https://github.com/puppetlabs/puppet-server/blob/86e2b6b79a9ad88c796579945d3d802cdc250359/documentation/configuration.markdown#puppetserverconf.